src/google/protobuf/compiler/csharp/csharp_generator.h

```diff
@@ -48,13 +48,16 @@ namespace csharp {
 // header.  If you create your own protocol compiler binary and you want
 // it to support C# output, you can do so by registering an instance of this
 // CodeGenerator with the CommandLineInterface in your main() function.
-class PROTOC_EXPORT Generator : public CodeGenerator {
+class PROTOC_EXPORT CSharpGenerator : public CodeGenerator {
```

Comment:
just curious: why rename the class? It's under the csharp namespace already, so strictly speaking this shouldn't be necessary.

Reply:
I did it for consistency with the other classes I was looking at - namely JavaGenerator and CppGenerator. Looking at PHP, Ruby, and Python though, they just have `Generator`.

@haberman any preference on this?